packages/vscode-extension/src/commands/Deploy.ts

```diff
@@ -0,0 +1,44 @@
+import SuiteCloudRunner from '../core/SuiteCloudRunner';
+import CommandsMetadataSingleton from '../service/CommandsMetadataSingleton';
+import { getRootProjectFolder, unwrapExceptionMessage } from '../util/ExtensionUtil';
+import { scloudOutput } from '../extension';
+import { MessageService } from '../service/MessageService';
+
+
+export default async function deploy() {
```

Comment:
Why a function? :(
We could create a class (with a clear interface) so all commands use the same